cmd/plume/prerelease.go

```diff
@@ -91,6 +94,19 @@ func init() {
 }
 
 func runPreRelease(cmd *cobra.Command, args []string) error {
+	getClient := func() (client *http.Client, err error) {
+		if gceJSONKeyFile != "" {
+			b, err := ioutil.ReadFile(gceJSONKeyFile)
+			if err != nil {
+				plog.Fatal(err)
+			}
+			client, err = auth.GoogleClientFromJSONKey(b)
```

Comment:
This will be needed for `plume release` also.  Maybe the similar code in platform/api/gcloud/api.go New() should be made a utility function in auth/google.go.